docs/quickstart/index.rst

```diff
@@ -22,7 +22,7 @@ Step 2: Run infrastructure.
 .. code-block::
    :caption: Run docker compose from the root of the quantum serverless project.
       cd quantum-serverless/
-      docker compose --profile full up
+      docker compose up
```

Comment:
Should we unpin the version above in quickstart if we do this? (see step 1)

Reply:
good call